decidim/decidim
# Change UI for endorsements

#### :tophat: What? Why?
In this PR we handle the UI changes regarding the Like mechanism. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13328

#### Testing
1. Make sure pipeline is green 
2. Apply the patch
3. Visit any of the Blog, Debate, Proposal component 
4. See the new UI & Functionality

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![image](https://github.com/user-attachments/assets/3d890a99-7640-4edc-88bc-55ed375d5ebf)

![image](https://github.com/user-attachments/assets/2d25cc11-512e-4b5f-b37c-a90f51182f1e)

:hearts: Thank you!
